include/unifex/scheduler_concepts.hpp

```diff
@@ -301,7 +308,44 @@ namespace _schedule_at {
         -> _result_t<TimeScheduler, TimePoint> {
       return static_cast<TimeScheduler&&>(s).schedule_at((TimePoint &&) tp);
     }
+
+    template <typename TimePoint>
+    constexpr sender<TimePoint> operator()(TimePoint tp) const {
+      return sender<TimePoint>{std::move(tp)};
+    }
   } schedule_at {};
+
+  template <typename TimePoint>
+  class _sender<TimePoint>::type {
+  public:
+    template <template <typename...> class Variant, template <typename...> class Tuple>
+    using value_types = Variant<Tuple<>>;
+
+    template <template <typename...> class Variant>
+    using error_types = Variant<std::exception_ptr>;
```

Comment:
This sender is reporting that it will only ever invoke set_error() with `std::exception_ptr` but the ultimate scheduler's schedule_at() sender might end up sending some other error types.

We'd either need to static_assert that its error_types are a subset of the error-types reported here, or apply an adapter that remapped other error types into a std::exception_ptr.

We can leave this for another PR though, and hopefully it would be solved properly by #16.